Tools/MGCB/BuildContent.cs

```diff
@@ -40,6 +40,30 @@ public List<string> ResponseFiles
             set { throw new InvalidOperationException(); }
         }
 
+        [CommandLineParameter(
+            Name = "autoinclude",
+            Flag = "a",
+            ValueName = "directory",
+            Description = "Autoincludes all the files for the specified directory.")]
+        // This property only exists for documentation.
```

Comment:
I don't know exactly when this started occurring... but we need to stop.

Implement the logic into the setter.